model/map-jpa/src/main/java/org/keycloak/models/map/storage/jpa/hibernate/dialect/JsonbPostgreSQL95Dialect.java

```diff
@@ -27,5 +27,6 @@ public JsonbPostgreSQL95Dialect() {
 //        registerFunction("->", new SQLFunctionTemplate(JsonbType.INSTANCE, "?1->?2"));
 //        registerFunction("->>", new SQLFunctionTemplate(StandardBasicTypes.STRING, "?1->>?2"));
 //        registerFunction("@>", new SQLFunctionTemplate(StandardBasicTypes.BOOLEAN, "?1@>?2::jsonb"));
+//        registerFunction("kc_hash", new SQLFunctionTemplate(StandardBasicTypes.BINARY, "sha256(?1::bytea)"));
```

Comment:
@mabartos @keycloak/storage-x during the process I noticed that all the `registerFunction()` methods in the constructor had been commented out. There was a conflict, which I resolved by adding the new method (L30) and commenting it out right afterwards. 

I suppose that for now this is expected, just wanted to specifically point it out as a reminder with regards to follow-ups.

Reply:
Thanks @Pepo48, as you say. It's expected and your solution is correct.